decisions/wc_troll_decisions.txt

```diff
@@ -0,0 +1,114 @@
+decisions = {	
+	form_the_empire_of_zul = {
+		is_high_prio = yes
+		only_playable = yes
+		only_independent = yes
+
+		potential = { 
+			culture_group = troll_group
+			any_demesne_title = {
+				OR = {
+					tier = KING
+					tier = EMPEROR
+				}
+			}
+			e_empire_of_zul = {
+				is_titular = yes
+			}
+			e_empire_of_zul = {
+				has_holder = no
+			}
+			is_alternate_start = no
+		}
+
+		allow = {
+			is_adult = yes
+			independent = yes
+			war = no
+			prestige = 5000
+			completely_controls_region = world_eastern_kingdoms_quelthalas_zulaman
+			completely_controls_region = world_eastern_kingdoms_lordaeron_the_hinterlands
+			completely_controls_region = world_eastern_kingdoms_azeroth_stranglethorn	
+			completely_controls_region = world_south_seas_zandalar
+			completely_controls_region = world_northrend_grizzly_hills
+			completely_controls_region = world_kalimdor_tanaris
+		}
+
+		effect = {
+			hidden_tooltip = {
+				primary_title = {
+					e_empire_of_zul = {
+						copy_title_laws = PREV
+						copy_title_history = PREV
+					}
+				}
+			}
+
+			custom_tooltip = {
+				text = create_empire_of_zul_tt
+				grant_title_no_opinion = e_empire_of_zul
+			}
+
+			hidden_tooltip = {
+
+				any_demesne_title = {
+					limit = {
+						tier = EMPEROR
```

Comment:
```suggestion
						can_be_conquered_trigger = yes
						tier = EMPEROR
```
`can_be_conquered_trigger = yes` prevents titles like the Legion, dragonflights from destroying.